content/blog/posts/visual-studio-2022-update.md

```diff
@@ -0,0 +1,277 @@
+```
+cmake --build build/windows --target Editor --config profile -- /maxcpucount:8 /p:CL_MPCount=16
+```
+
+The equivalent options in the Visual Studio 2022 IDE can be set via following IDE options
+* Setting the maximum number of parallel project builds value in the `Tools -> Options -> Projects and Solutions -> Build and Run` menu.  
+![Build and Run](/images/blog/vs2022/vs2022-parallel-project-builds-8.png)
+* Setting the maximum concurrent C++ compilations option in the `Tools -> Options -> Projects and Solutions -> VC++ Project Settings` menu.  
+![VC++ Project Settings](/images/blog/vs2022/vs2022-concurrent-compiliations-16.png)
+
+This can to help alliviate any memory pressure that is caused by building the O3DE.
+
+### VS2022(v143) is stricter at triggering warnings than VS2019(v142)
+The second notable differences is that Visual Studio 2022 is better at triggering compiler warnings than VS2019.  
+So occasionally code that would compile successfully in VS2019, may fail with a compiler warning in VS2022.  
+The triggered warnings are generally around [unsigned/signed](https://learn.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-3-c4018?view=msvc-170) comparison mismatches and [usused variables](https://learn.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-4-c4189?view=msvc-170).
+
+
+
+## Useful build tips
+
+### Building against the VS2022 v142 toolset using VS2022
+
+One lesser known trick to build against multiple Microsoft build tools toolsets(v143, v142, etc...) without the need to have multiple versions of Visual Studio installed involves specifying the toolset option for Visual Studio on the command line when configuring with CMake.  
+
+First the user should make sure they have the latest `MSVC v142 - VS 2019 C++ x64/x86 build tools` component installed via the Visual Studio Installer.  
+![Visual Studio Installer Individual Components](/images/blog/vs2022/vs2022-installer-v142-toolset.png)
+
+Once it is verified that latest `MSVC v142` toolset is installed, the user can configure the toolset using either the CMake GUI or the CMake CLI via the `-T<toolset>` option which is documented on the CMake website as part of the [CMAKE_GENERATOR_TOOLSET](https://cmake.org/cmake/help/v3.23/variable/CMAKE_GENERATOR_TOOLSET.html#visual-studio-toolset-selection) variable documentation.  
+To use the the latest v142 toolset which is 14.29 at this time, the following value for the `-T` option should be supplied:
+```
+-Tv142
+```
+
+
+#### Configuring the CMake Gui to use the v142 toolset using VS 2022
+
+Using the CMake GUI to build against the VS2019 v142 toolset is as simple as specifying v142 to the toolset parameter
+![CMake GUI Visual Studio Toolset](/images/blog/vs2022/cmake-gui-14-29-toolset.png)
```

Comment:
`![CMake GUI Visual Studio Toolset](/images/blog/vs2022/cmake-gui-14-29-toolset.png)`
Change to
`{{< image-width src="/images/blog/vs2022/cmake-gui-14-29-toolset.png" width="100%" alt="CMake GUI Visual Studio Toolset" >}}`